mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -43,18 +43,20 @@
 		if (!this.isPlacedOnMap() || !this.isAlive() || this.isDying()) return ret;
 		if (this.isDiscovered() == false) return ret;
 		if (this.isHiddenToPlayer()) return ret;
-
-		if (this.m.IsActingEachTurn && !this.m.IsTurnDone && this.isWaitActionSpent())
```

Comment:
Now the tooltip is broken for characters like donkeys (presumably the same as eggs or phylacteries)

![image](https://github.com/user-attachments/assets/00e9fe95-5751-4ae1-8198-fd5b0c17f45d)

Reply:
I've pushed a fix for this.

Reply:
Works now